front-end/src/routes/AppRouter.jsx

```diff
@@ -149,6 +150,14 @@ const RouterWrapper = () => {
             </PrivateRoute>
           }
         />
+        <Route
+          path='/app/student-subscriptions'
```

Comment:
Can we create only 1 route for student and teacher subscriptions and then check by user role for render right component?